testsuite/features/secondary/proxy_as_pod_basic_tests.feature

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped:
+# * if there is no proxy ($proxy is nil)
+# * if there is no salt minion ($sle_minion is nil)
+# * if there is no scope @scope_containerized_proxy
+
+@scope_containerized_proxy
+@proxy
+@sle_minion
+Feature: Register and test a Containerized Proxy
```

Comment:
Sometime you write `Containerized` with a capital C and sometimes without. This should be unified.

Reply:
:D Ok, I will take a look